dxr/vcs.py

```diff
@@ -84,9 +84,9 @@ def display_rev(self, path):
 class Mercurial(Vcs):
     def __init__(self, root):
         super(Mercurial, self).__init__(root, 'hg')
-        hgext = join(dirname(dxr.__file__), 'hgext', 'previous_revisions.py')
+        hgext = resource_filename(__name__, 'hgext/previous_revisions.py')
```

Comment:
We probably shouldn't use `__name__` here. Take a look at https://pythonhosted.org/setuptools/pkg_resources.html#basic-resource-access. This becomes "dxr.vcs", which fortunately works because `vcs` is a module. But someday, when `vcs` grows and becomes a package, this will quit working. The most durable thing would probably be to import dxr and then use `dxr.__name__` here. Or we could just hard-code 'dxr', which would work unless we renamed the whole project.